code/modules/mob/living/carbon/human/human.dm

```diff
@@ -395,7 +395,7 @@
 	. = ..()
 	if(!. && (injection_flags & INJECT_TRY_SHOW_ERROR_MESSAGE) && user)
 		var/obj/item/bodypart/the_part = get_bodypart(target_zone || user.zone_selected)
-		to_chat(user, span_alert("There is no exposed flesh or thin material on [p_their()] [the_part.name]."))
+		to_chat(user, span_alert("There is no exposed flesh or thin material on [isnull(the_part) ? "them" : "[p_their()] [the_part.name]"]."))
```

Comment:
why would this be null?

Reply:
`get_bodypart(target_zone || user.zone_selected)` returns null for precise targetting.
let me double check as to why

Reply:
![image](https://user-images.githubusercontent.com/16159590/148670056-08ce4bc3-637d-4c9f-bf93-0808306a20e8.png)
`get_bodypart()` loops through `bodyparts` which don't contain any of the precise zones